stable/yugaware/values.yaml

```diff
@@ -176,3 +176,36 @@ additionalAppConf:
 ## to modify this unless you are using helm template command i.e. GKE
 ## app's deployer image against a Kubernetes cluster >= 1.21.
 # pdbPolicyVersionOverride: "v1beta1"
+
+## kube-state-metric support
+## Custom endpoint has higher priority than installation. 
+kubeStateMetric:
```

Comment:
```suggestion
## kube-state-metrics support
## Custom endpoint has higher priority than installation. 
kubeStateMetrics:
```
To make it consistent with the project name. This will need a change at all the places where it is used.